.github/profile/README.md

```diff
@@ -0,0 +1,23 @@
+## Welcome to our power network full of...
+
+<img src="phlowers_logotxt.png" width="200" height="200" style="display: block; margin: 0 auto"/>
```

Comment:
I would add an alt attribute here
Also, it's hard to read the text "phlowers" when using Github with a dark theme because then the first three letters are displayed in dark blue against dark grey. Adding a white background would help make it more readable.

Reply:
Good point for the logo. The next one is with white background to avoid contrast issues